packages/api/cli/src/electron-forge-start.ts

```diff
@@ -2,15 +2,31 @@ import path from 'path';
 
 import { api, StartOptions } from '@electron-forge/core';
 import { ElectronProcess } from '@electron-forge/shared-types';
+import chalk from 'chalk';
 import program from 'commander';
 import fs from 'fs-extra';
+import semver from 'semver';
+// eslint-disable-next-line node/no-unpublished-import
+import updateNotifier from 'update-notifier';
 
 import './util/terminate';
 import workingDir from './util/working-dir';
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const metadata = require('../package.json');
 (async () => {
   let commandArgs = process.argv;
   let appArgs;
+  const notifier = updateNotifier({
+    pkg: {
+      name: metadata.name,
+      version: metadata.version,
+    },
+    updateCheckInterval: 1000 * 60 * 60,
+  });
+  if (notifier.update && semver.lt(notifier.update.current, notifier.update.latest)) {
+    console.log(`Update available: ${chalk.dim(`${notifier.update?.current}`)} -> ${chalk.green(`${notifier.update?.latest}`)}`);
```

Comment:
issue(blocking): I don't think this is exactly what we want here.

I think a custom log message is necessary for our monorepo design as we want to provide users with a way to update all their packages at a glance, but it seems like the custom message here simply logs the new version number without additional context?

Reply:
Additionally, I'd like to manually generate the little box that updatenotifier uses by default :)